### ISSUE TITLE
[Keyboard Navigation-SQL Pref Main]: 'Import' button present in the Tasks Dropdown is not keyboard accessible.

### ISSUE THREAD

**Issue by PiJoCoder** [microsoft/SqlNexus#371]
Impact on a Person with Disability (PWD):
When interactive components are not accessible, assistive technology users are blocked, which leads to everything from inaccessible content, features, and functionality, up to entire applications or sites.

Test Environment:​
OS: Windows 11 [Version 22H2 (OS Build 22621.2283)].

Desktop application: SQL Nexus.

Pre-Requisites:
Install SQLNexus application.

Repro Steps:
Step 1: Open the SQL Nexus app on the desktop.
Step 2: Connect to server Dialog gets displayed. Connect to the server by invoking connect button.
Step 3: 'SQL Pref Main' page getting displayed.
Step 4: Navigate to import button present in the Tasks dropdown.
Step 5: Verify whether 'Import' button present in the Tasks Dropdown is keyboard accessible or not.
 
Actual Result:
'Import' button present in the Tasks Dropdown is not keyboard accessible.
Current Observation: When focus lands on the Import control, no action gets invoked upon pressing enter/space. But through mouse we are able to access it.
Screen reader observation: When focus lands on the Import button, Narrator is narrating as 'Import link' and on invoking enter/space key no action is getting performed and screen reader is remaining silent.

Expected Result:
'Import' button present in the Tasks Dropdown should be keyboard accessible.
Ex When focus lands on the Import control and upon pressing enter/space user should be accessible the Import control.

Suggestive Fix:
In most cases, using standard HTML controls will ensure keyboard accessibility.
If functionality requires going beyond standard HTML controls, use WAI-ARIA authoring practices to ensure that keyboard functionality is maintained.

**Comment by PiJoCoder** [comment]
1711596

**Comment by PiJoCoder** [comment]
Fixed by PR #365
